src/stackComponent/src/Stack.js

```diff
@@ -0,0 +1,49 @@
+import React, { memo, forwardRef } from 'react'
+import PropTypes from 'prop-types'
+import { spacing, dimensions, position, layout } from 'ui-box'
+import { Pane } from '../../layers'
+
+const Stack = memo(
+  forwardRef(function Stack(props, ref) {
+    const { className, children, space, align, ...restProps } = props
+
+    let justifyItems = align
+    if (align === 'left') justifyItems = 'start'
+    if (align === 'right') justifyItems = 'end'
+
+    return (
+      <Pane
```

Comment:
Could we actually swap this `<Pane />` out for a `<Box />` from `ui-box`? We are trying to reduce inter-component dependencies